app/user/rest.py

```diff
@@ -440,32 +440,18 @@ def send_already_registered_email(user_id):
 def send_support_email(user_id):
     data, errors = support_email_data_schema.load(request.get_json())
 
-    API_URL = current_app.config['FRESH_DESK_API_URL']
-    API_KEY = current_app.config['FRESH_DESK_API_KEY']
-
-    ticket = {
-        'product_id': 61000000046,
-        'subject': data["support_type"] if "support_type" in data else "Support Request",
-        'description': data['message'],
-        'email': data["email"],
-        'priority': 1,
-        'status': 2,
-    }
-
-    response = requests.post(
-        "{}/api/v2/tickets".format(API_URL),
-        json=ticket,
-        auth=HTTPBasicAuth(API_KEY, "x")
-    )
+    data['tags'] = ['z_skip_opsgenie', 'z_skip_urgent_escalation']
+    try:
+        user = get_user_by_email(data['email'])
+        has_live_services = any([not s.restricted for s in user.services])
+        if has_live_services:
+            data['tags'] = []
+    except NoResultFound:
+        pass
```

Comment:
Are we the ones who wrote the `get_user_by_email` function? It seems weird to raise an exception because no results were returned.

Reply:
Nope, introduced here https://github.com/cds-snc/notification-api/commit/e055590b0730acb810cba034456c5306671b195d

`.one()` raises an exception, otherwise, we need to deal with a potential null object. I'm fine catching the exception

Reply:
Yeah, it seems prevalent in the Python community to use exceptions for control flow.